lib/cli/output/show.js

```diff
@@ -11,6 +16,47 @@ const stopSpinner = () => {
   spinner.stop();
 };
 
+const showScoreLess = async (scoreboard) => {
+  const file = '/tmp/autolabjs/scoreboard';
+  try {
+    await fs.outputFile(file, scoreboard);
+    spawn('cat /tmp/autolabjs/scoreboard | less -r', {
```

Comment:
`exec()` spawns a shell then executes the command within that shell, buffering any generated output to the callback provides. That is the reason I have used `spwan()` instead.